--- user guide EN 1.0/core_plugins.tex	2009-11-30 19:25:27.242714938 +0100
+++ user guide EN 1.3/core_plugins.tex	2009-11-28 23:05:58.947670937 +0100
@@ -6,12 +6,8 @@
 % comment out the following line:
 %\updatedisclaimer
 
-QGIS currently contains 17 core plugins that can be loaded using the Plugin Manager.
-Table \ref{tab:core_plugins} lists each of the core plugins along with a
-description of their purpose and the toolbar-icon.\footnote{The MapServer
-Export Plugin and the Plugin Installer Plugin are external Python Plugins,
-but they are part of the QGIS sources and automatically loaded and selectable 
-inside the QGIS Plugin Manager.}
+QGIS \CURRENT contains 22 core plugins that can be loaded using the Plugin 
+Manager (see Table \ref{tab:core_plugins}).
 
 % minipage is needed to appear the footnote under the table
 % SH
@@ -31,10 +27,19 @@
 \hline 
 \includegraphics[width=0.6cm]{copyright_label}
  & Copyright Label \index{plugins!copyright}& Draws a copyright label with information\\
-\hline 
+\hline
+\includegraphics[width=0.6cm]{diagram_overlay}
+ & Diagram Overlay \index{plugins!diagram}& Place charts (pie or bar) or proportional symbols over vector layers\\
+\hline
 \includegraphics[width=0.6cm]{dxf2shp_converter}
  & DXF2Shape Converter \index{plugins!DXF2Shape}& Converts from DXF to SHP file format\\
 \hline
+\includegraphics[width=0.6cm]{evis_icon}
+ & eVis & Event Visualization Tool \\
+\hline
+\includegraphics[width=0.6cm, height=0.6cm]{ftoolslogo}
+ & fTools \index{plugins!ftools}& A suite of analysis, geometry, geoprocessing, and research tools\\
+\hline
 \includegraphics[width=0.6cm]{gps_importer}
  & GPS Tools \index{plugins!gps}& Tools for loading and importing GPS data\\
 \hline
@@ -44,12 +49,13 @@
 \includegraphics[width=0.6cm]{georeferencer}
  & Georeferencer \index{plugin!georeferencer} & Adding projection info to Rasterfiles\\
 \hline
-\includegraphics[width=0.6cm]{grid_maker}
- & Graticule Creator \index{plugins!graticule}& Create a latitude/longitude grid and save as a shapefile\\
-\hline
 \includegraphics[width=0.6cm]{interpolation}
 & Interpolation plugin \index{plugins!Interpolation}& Interpolation on base of vertices of a vector layer\\
 \hline
+\includegraphics[width=0.6cm]{raster_terrain}
+& Raster Terrain Modelling \index{plugins!Raster Terrain Modelling}& Compute slope, aspect,
+ruggedness and total curvature of DEMs\\
+\hline
 \includegraphics[width=0.6cm]{mapserver_export}
 & MapServer Export Plugin \index{plugins!MapServer Export}& Export a saved QGIS project file to a MapServer map file \\
 \hline
@@ -60,6 +66,12 @@
  & OGR Layer Converter \index{plugins!OGR converter} & Translate vector
 layers between OGR suported formats\\
 \hline
+\includegraphics[width=0.6cm]{osm_icon}
+ & OpenStreetMap & Visualize and edit OpenStreetMap data \\
+\hline
+\includegraphics[width=0.6cm]{oracle_raster}
+ & Oracle Georaster \index{plugins!georaster}& Access Oracle Spatial GeoRasters\\
+\hline
 \includegraphics[width=0.6cm]{plugin_installer}
  & Plugin Installer \index{plugins!Plugin Installer} & Downloads and installs QGIS python plugins\\
 \hline
@@ -81,7 +93,3 @@
 \end{minipage}
 
 \normalsize
-
-\begin{Tip}\caption{\textsc{Plugins Settings Saved to Project}}\index{plugins settings}
-\qgistip{When you save a .qgs project, any changes you have made to NorthArrow, ScaleBar and Copyright plugins will be saved in the project and restored next time you load the project.}
-\end{Tip}
