*** /home/jrm/Documents/QGIS/english/user_guide_1.5/getting_started.tex	2010-08-14 13:25:15.764791597 +0200
--- /home/jrm/Documents/QGIS/english/user_guide_1.6/getting_started.tex	2010-11-29 21:39:56.903084790 +0100
***************
*** 26,31 ****
--- 26,38 ----
  The installation instructions are also distributed with the \qg source
  code.
  
+ \minisec{Installation on external media}
+ 
+ QGIS allows to define a --configpath option that overrides the default path 
+ (e.g. ~/.qgis under Linux) for user configuration and forces QSettings to use 
+ this directory, too. This allows users to e.g. carry a QGIS installation on a 
+ flash drive together with all plugins and settings. 
+ 
  \section{Sample Data}\label{label_sampledata}
  \index{data!sample} 
  
