*** /home/jrm/Documents/QGIS/english/user_guide_1.5/print_composer.tex	2010-09-19 11:15:03.602635279 +0200
--- /home/jrm/Documents/QGIS/english/user_guide_1.6/print_composer.tex	2010-11-29 21:39:56.901084774 +0100
***************
*** 124,130 ****
  You can add multiple elements to the composer. It is also possible to have
  more than one map view or legend or scalebar in the print composer canvas.
  Each element has its own properties and in the case of the map, its own
! extent.
  
  \section{Adding a current QGIS map canvas to the Print Composer}
  
--- 124,131 ----
  You can add multiple elements to the composer. It is also possible to have
  more than one map view or legend or scalebar in the print composer canvas.
  Each element has its own properties and in the case of the map, its own
! extent. If you want to remove an elements from the composer canvas. you can 
! do that with the \keystroke{delete} or the \keystroke{backspace} key.
  
  \section{Adding a current QGIS map canvas to the Print Composer}
  
***************
*** 281,288 ****
  
  \begin{itemize}[label=--]
  \item The \textbf{Label} dialog offers to add text labels to the composer
! canvas. You can select font and fontcolor for the text and it is possible to
! define a text margin im mm.
  \end{itemize}
  
  \minisec{General options dialog}
--- 282,289 ----
  
  \begin{itemize}[label=--]
  \item The \textbf{Label} dialog offers to add text labels to the composer
! canvas. You can define the horizontal and vertical alignment, select font 
! and fontcolor for the text and it is possible to define a text margin im mm.
  \end{itemize}
  
  \minisec{General options dialog}
***************
*** 498,506 ****
  
  \begin{itemize}[label=--]
  \item The \textbf{Table} dialog allows to select the vector layer and columns of
! the attribute table. You can define the maximum number of rows to be displayed
  and if attributes are only shown for visible features of the current composer
! canvas. Additionally you can define the grid characteristics of the table and
  the header and content font.
  \end{itemize}
  
--- 499,510 ----
  
  \begin{itemize}[label=--]
  \item The \textbf{Table} dialog allows to select the vector layer and columns of
! the attribute table. Attribute columns can be sorted and you can define to show 
! its values ascending or descending. 
! \item You can define the maximum number of rows to be displayed
  and if attributes are only shown for visible features of the current composer
! canvas. 
! \item Additionally you can define the grid characteristics of the table and
  the header and content font.
  \end{itemize}
  
