*** /home/jrm/Documents/QGIS/english/user_guide_1.5/working_with_projections.tex	2010-09-11 20:59:05.957629034 +0200
--- /home/jrm/Documents/QGIS/english/user_guide_1.6/working_with_projections.tex	2010-11-29 21:39:56.910084848 +0100
***************
*** 102,108 ****
  defined CRS.
  
  The \tab{Coordinate Reference System} tab of the \dialog{Project Properties}
! dialog contains four important components as shown in Figure
  \ref{fig:projections} and described below.
  
  \begin{figure}[ht]
--- 102,108 ----
  defined CRS.
  
  The \tab{Coordinate Reference System} tab of the \dialog{Project Properties}
! dialog contains five important components as shown in Figure
  \ref{fig:projections} and described below.
  
  \begin{figure}[ht]
***************
*** 129,135 ****
  Enter the identifier and click on \button{Find}. Use the \checkbox{Hide
  deprecated CRSs} checkbox to show only the currently valid projections.
  \item \textbf{Recently used CRS} - if you have certain CRS that you frequently
! use in your everyday GIS work, these will be displayed as 'quick access' buttons
  at the bottom of the Projection Dialog. Click on one of these buttons to select
  the associated CRS.
  \end{enumerate}
--- 129,135 ----
  Enter the identifier and click on \button{Find}. Use the \checkbox{Hide
  deprecated CRSs} checkbox to show only the currently valid projections.
  \item \textbf{Recently used CRS} - if you have certain CRS that you frequently
! use in your everyday GIS work, these will be displayed in the table
  at the bottom of the Projection Dialog. Click on one of these buttons to select
  the associated CRS.
  \end{enumerate}
