--- user guide EN 1.0/plugins_decorations.tex	2009-11-30 19:25:27.244724455 +0100
+++ user guide EN 1.3/plugins_decorations.tex	2009-11-28 23:05:58.949671372 +0100
@@ -31,9 +31,8 @@
 \item Click \button{OK} 
 \end{enumerate}
 
-In the example above, the first line is in bold, the second (created using
-\textless br\textgreater) contains a copyright symbol, followed by our company name in
-italics.
+In the example above (default) places a copyright symbol followed by the date in the 
+lower right hand corner of the map canvas.
 
 \subsubsection{North Arrow Plugin}
 
@@ -80,3 +79,8 @@
    \includegraphics[clip=true, width=8cm]{scale_bar_dialog}
 \end{center}  
 \end{figure}
+
+\begin{Tip}\caption{\textsc{Plugins Settings Saved to Project}}\index{plugins settings}
+\qgistip{When you save a .qgs project, any changes you have made to NorthArrow, ScaleBar and Copyright plugins will be saved in the project and restored nexttime you load the project.}
+\end{Tip}
+
