*** /home/jrm/Documents/QGIS/english/user_guide_1.5/help_and_support.tex	2010-08-27 19:08:52.999240360 +0200
--- /home/jrm/Documents/QGIS/english/user_guide_1.6/help_and_support.tex	2010-11-29 21:39:56.911084856 +0100
***************
*** 56,61 ****
--- 56,74 ----
  subscribe to this list at:\\
  \url{http://lists.osgeo.org/mailman/listinfo/qgis-release-team}
  
+ \minisec{qgis-tr}
+ 
+ This list deals with the translation efforts. If you like to work on the translation 
+ of the manuals or the graphical user interface (GUI), this list is a good starting 
+ point to ask your questions.
+ 
+ \minisec{qgis-edu}
+ 
+ This list deals with QGIS education efforts. If you like to work on qgis education 
+ materials, this list is a good starting point to ask your questions.
+ 
+ \url{http://lists.osgeo.org/mailman/listinfo/qgis-edu}
+ 
  \minisec{qgis-psc}
  This list is used to discuss Steering Committee issues related to overall
  management and direction of Quantum GIS. You can subscribe to this list at:\\
